src/iobuf.ml

```diff
@@ -192,6 +192,11 @@ let set_bounds_and_buffer_sub ~pos ~len ~src ~dst =
   then dst.buf <- src.buf
 ;;
 
+(* use a unique identifier to refer to set_bounds_and_buffer_sub
+ * so that Expert.set_bounds_and_buffer_sub can refer to it
+ * explicitly. *)
+let iobuf_set_bounds_and_buffer_sub = set_bounds_and_buffer_sub
+
```

Comment:
Is it necessary to introduce an alias here? Wouldn't simply saying `let set_bounds_and_buffer_sub = set_bounds_and_buffer_sub` below work?